lib/module_importer.js

```diff
@@ -63,18 +63,28 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
     var moduleName = fragments[0];
     var relativePath = fragments.slice(1).join("/");
     var jsFile = eyeglass.modules.access(moduleName, isRealFile ? prev : root);
+    var sassDir;
 
-    if (jsFile && jsFile.main) {
-      // is eyeglass module
+    if (jsFile && jsFile.eyeglass.sassDir) {
+      sassDir = jsFile.eyeglass.sassDir;
+    } else if (jsFile && jsFile.main) {
       var mod = require(jsFile.main)(eyeglass, sass);
-      var sassDir = mod.sassDir;
-      if (!sassDir) {
-        importUtils.fallback(uri, prev, done, function() {
-          done(new Error("sassDir is not specified in " + jsFile.main));
-        });
-        return;
-      }
+      sassDir = mod.sassDir;
+    }
+
+    if (jsFile && !(sassDir || isRealFile)) {
```

Comment:
instead of...

```js
if (jsFile && jsFile.eyeglass.sassDir) {
   ...
} else if (jsFile && jsFile.main) { ... }

if (jsFile && !(sassDir || isRealFile)) { ... }
```

consider...

```js
if (jsFile) {
  if(jsFile.eyeglass.sassDir) {
    ...
  } else if (jsFile.main) { ... }
  
  if (!(sassDir || isRealFile)) { ... }
}
```